server.js

```diff
@@ -18,8 +19,65 @@ import env from "./config/env.generated.json";
 const defaultPort = 3000;
 const PORT = env.PORT || process.env.PORT || defaultPort;
 
+/**
+ * Some app security settings
+ */
+
+var securityHeaders = {
+  directives: {
+    defaultSrc: [
+      `\'none\'`
+    ],
+    scriptSrc: [
+      `\'self\'`,
+      `\'unsafe-inline\'`
+    ],
+    fontSrc: [
+      `\'self\'`,
+      `https://code.cdn.mozilla.net`
+    ],
+    styleSrc: [
+      `\'self\'`,
+      `\'unsafe-inline\'`,
+      `https://code.cdn.mozilla.net`
+    ],
+    imgSrc: [
+      `\'self\'`,
+      `\'unsafe-inline\'`,
+      `data:`,
+      `*`
+    ],
+    connectSrc: [
+      `\'self\'`,
+      env.PULSE_API || `https://network-pulse-api-staging.herokuapp.com/`
+    ]
+  },
+  reportOnly: false,
+  browserSniff: false
+};
+
+app.use(helmet.contentSecurityPolicy(securityHeaders));
+
+app.use(helmet.xssFilter({
+  setOnOldIE: true
```

Comment:
Microsoft no longer supports old IE, as of January 2016 it only officially supports IE11 and EDGE, so I'm pretty okay taking this out. If you're on an old version of IE, our website is the least of your problems. @cadecairos opinions?